apps/ledger-live-desktop/src/renderer/modals/SignTransaction/index.tsx

```diff
@@ -13,6 +14,9 @@ const SignTransactionModal = ({ stepId }: Props) => {
     stepId: stepId || "summary",
     error: undefined as Error | undefined,
   });
+  useEffect(() => {
+    ipcRenderer.send("ready-to-show", {});
+  }, []);
```

Comment:
It would be better to make our own `show` handler on main because `ready-to-show` will trigger a deeplink for windows and linux.

```ts
ipcMain.on("show", () => {
  const w = getMainWindow();
  if (w) {
    show(w);
  }
});
```

Also it would be good to do the logic in the wallet-api part so that we show LLD on any message that requires an action from the user